### PR TITLE
Create onboarding.md

### PR DESCRIPTION
**Pull request by arfintechpy**
Creating an onboarding section, with the purpose of presenting a list of assets at a glance for newcomers.

### CODE REVIEW

**Comment by kjw000** [comment]
Thanks for fixing the conflicts.  Agree this looks good and we can add on to it.

**Comment by kjw000** [comment]
@arfintechpy - can you and Henk @HenkKodde  figure out where to link this document.  It should be somewhere at the top of the documentation chain and also in the FAQ.

**Comment by arfintechpy** [comment]
Yes, will keep working on improving it with Henk. We've been discussing the content together.

File: onboarding.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- **Mojaloop Hub**: The Hub is the container of all the core central services of Mojaloop. The following are the options available to work with the Hub:
+  - **Public sandbox:** If you want to test some transaction flows without going through the process of deploying the Hub, there is a sandbox already setup for testing. The corresponding links to it are contained in the Postman collections detailed below.
+  - **Deployment guide:** If you wish to deploy Mojaloop on your own environment, please find the [Deployment Guide](../deployment-guide/). 
+- **FSP Simulator:** This acts as a mock FSP to simulate the Payer and Payee roles in a transaction. Building and configurations details can be found in the [Github simulator repository](https://github.com/mojaloop/simulator).
+- **Postman:** Postman is an application to send requests and receive responses. The following resources can be imported into a Postman client to interact with the Hub:
+  - A **collection** containing a set of requests to run several operations: [Postman collection](https://raw.githubusercontent.com/mojaloop/postman/master/Mojaloop.postman_collection.json).
```
Comment (mdebarros) [inline onboarding.md]: Note that we have removed this collection as they are not being maintained. Please have a look at the most recent updates to the https://github.com/mojaloop/postman repo for more information.

File: onboarding.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - An **environment** containing the values of the corresponding input parameters, pointing to the public Mojaloop sandbox: [Postman environment](https://raw.githubusercontent.com/mojaloop/postman/master/environments/MojaloopLocal.postman_environment.json).
```
Comment (mdebarros) [inline onboarding.md]: Same comment as above. This has been removed as it was not being maintained.

**Comment by mdebarros** [comment]
Closing this request as @arfintechpy will be creating a new PR since the branch/fork was deleted.

**Pull request closed without merging**